photon-core/src/main/java/org/photonvision/common/configuration/NeuralNetworkModelManager.java

```diff
@@ -239,7 +241,8 @@ public static NeuralNetworkModelManager getInstance(boolean reset) {
 
     public enum Family {
         RKNN(".rknn"),
-        RUBIK(".tflite");
+        RUBIK(".tflite"),
+        APPLE(".mlmodel");
```

Comment:
I know we've used this "apple" nomenclature a lot. Is CoreML a more descriptive name?